src/main/java/fi/okm/jod/yksilo/service/koski/KoskiService.java

```diff
@@ -159,4 +172,25 @@ private static LocalDate getLocalDate(JsonNode nimet, String path) {
     JsonNode node = readJsonProperty(nimet, path);
     return node != null && node.isTextual() ? LocalDate.parse(node.asText()) : null;
   }
+
+  @Cacheable(value = "osaamisetIdentified", key = "'user_' + #user.id")
+  @Transactional(readOnly = true)
+  public List<KoulutusDto> getOsaamisetIdentified(JodUser user, List<UUID> uuids) {
+    if (uuids.size() > (Limits.KOULUTUSKOKONAISUUS * Limits.KOULUTUS_PER_KOKONAISUUS)) {
```

Comment:
This validation should be implemented at the controller layer, using bean validation (`@Size(max=....)), if at all since since the product is 1 000 000 and one can not pass that many uids in the request anyway.